### PR TITLE
Node.js / Next.js compatibility

### PR DESCRIPTION
**Pull request by bpierre**
Fixes https://github.com/aragon/use-wallet/issues/22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.4.4",
   "author": "Aragon Association <legal@aragon.org>",
   "license": "AGPL-3.0",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
```
Comment (sohkai) [inline package.json]: Ah nice, is it still expected to use `main` as the commonJS export in package.jsons? I remember there might've been another key being proposed, but haven't really haven't followed any discussion.

File: rollup.config.js [hunk 2]
```diff
@@ -23,6 +32,7 @@ export default {
       preferBuiltins: false,
     }),
     commonjs(),
+    nodePolyfills(),
```
Comment (sohkai) [inline rollup.config.js]: Does this increase the bundle size a lot? I wonder if this is just something we should have in the commonJS export?

Comment (bpierre) [reply rollup.config.js]: IIRC this was meant to fix an issue with one of the connectors trying to load a Node module, but I can’t reproduce the issue so I removed it for now, but it didn’t seem to impact the size much.

File: package.json (hunk 1 above)
Comment (sohkai) [reply package.json]: Sounds good!

**Pull request merged**